spec/lib/tenejo/preflight_spec.rb

```diff
@@ -26,7 +26,7 @@
     end
   end
   context "a file that isn't a csv " do
-    let(:graph) { described_class.read_csv("spec/fixtures/images/cat.jpg", "tmp/uploads") }
+    let(:graph) { described_class.read_csv("spec/fixtures/images/tiny.jpg", "tmp/uploads") }
     it "returns a fatal error" do
       expect(graph[:fatal_errors]).to eq ["Could not recognize this file format: Invalid byte sequence in UTF-8 in line 1."]
```

Comment:
This doesn't seem like the friendliest error message ever - we could just pass the full job object (or the filename) and you could do something like
```
"Could not interpret file format for: #{uploaded_filename}"
```

Or, just 
```
"File format or encoding unrecognized"
```

Reply:
We could also just check that the first line "appears" to have at least 4(?) and less than 30(?) commas - ID,Type,Title,visibility,rights_statement

If the first row seems to have more or fewer commas, it's probably not a valid header??

Also, we could put a *.csv restriction on the file upload input.